blueprint/index.md

```diff
@@ -1 +1,210 @@
+
+### AWS account
+* An administrator account with permissions to access the following services:
+  * AWS Identity and Access Management (IAM)
+  * Amazon API Gateway
+  * Amazon DynamoDB
+  * AWS Lambda
+* AWS credentials. For more information about setting up your AWS credentials on your local machine, see [About credential providers](https://docs.aws.amazon.com/sdkref/latest/guide/creds-config-files.html "Opens the About credential providers page") in AWS documentation.
+* AWS CLI. For more information about installing the AWS CLI on your local machine, see [About credential providers](https://aws.amazon.com/cli/ "Opens the About credential providers page") in the AWS documentation.
+
+## Implementation steps
+
+1. Clone the GitHub repository
+
+### Clone the GitHub repository
+Clone the GitHub repository [ionic-mobile-open-messaging-blueprint](https://github.com/GenesysCloudBlueprints/ionic-mobile-open-messaging-blueprint "Opens the ionic-mobile-open-messaging-blueprint repository). The repository folder includes the sample configuration settings and folder structure for the front-end and back-end of the application. 
+
+### Create an OAuth client in Genesys Cloud
+
+1. In Genesys Cloud, navigate to **Admin** > **OAuth** and click **Add Client**.
+2. Create an OAuth client with the following options:
+   1. Enter a name for the client and select **Client Credentials** as the Grant Type.
+   2. Click the **Roles** tab and select the role that has the following permissions:
+      1. Analytics > conversationDetail > View
+      2. Conversation > message > All Permissions
+3. Click **Save** and note the **Client ID** and **Client Secret** fields to use in the AWS Lambda setup.
+
+### Create the Amazon DynamoDB tables
+
+1. Create a table for Open messaging transcript. 
+   This is a persistent storage for the chat transcript that the Genesys Cloud writes to when an agent replies to a customer. Add the following columns to the table:
+   1. id
+      * type - String
+      * Partition Key
+   2. channelTime
+      * type - String
+      * Sort Key
+2. Create a table for the Notifications. This table is for pushing notifications to the app. The table is used only with the API. If you want Genesys Cloud to write to this table, then you must incorporate a data action to call the API. This table is optional only if you have the Notifications tab in the app. Add the following columns to the table:
+   1. id
+      * type - String
+      * Partition Key
+   2. channelTime
+      * type - String
+      * Sort Key
+
+### Create an AWS S3 bucket
+To store any media sent from the Ionic front-end app, create an Amazon S3 bucket storage. The S3 bucket must have public read access for the objects. Ensure that you implement the security measures that are in line with your company.
+
+### Create a function using AWS Lambda
+
+1. Create a function using AWS Lambda in your AWS account.
```

Comment:
probable need to reference NodeJS here.  I missed it mine too.  Something like: "Create a function using a NodeJS AWS Lambda in your AWS account."

Reply:
added.